examples/experimental/persistent-volume-provisioning/README.md

```diff
@@ -81,6 +81,28 @@ parameters:
 * `type`: `pd-standard` or `pd-ssd`. Default: `pd-ssd`
 * `zone`: GCE zone. If not specified, a random zone in the same region as controller-manager will be chosen.
 
+#### GLUSTERFS
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: StorageClass
+metadata:
+  name: slow
+provisioner: kubernetes.io/glusterfs
+parameters:
+  endpoint: "glusterfs-cluster"
+  resturl: "http://127.0.0.1:8081"
+  restauthenabled: "true"
+  restuser: "admin"
+  restuserkey: "password"
+```
+
+* `endpoint`: `glusterfs-cluster` is the endpoint/service name which includes GlusterFS trusted pool IP addresses and this parameter is mandatory.
+* `resturl` : Gluster REST service url which provision gluster volumes on demand. The format should be `IPaddress:Port` and this is a mandatory parameter for GlusterFS dynamic provisioner.
```

Comment:
This will be confusing. There is going to be a Gluster REST service to communicate to glusterd and then there is Heketi, the RESTful volume manager for Gluster.  I think all of these values should be renamed to heketiXXX instead of restXXX to avoid confusion.